backend/server/project.go

```
@@ -136,17 +139,114 @@ func (app *Application) DeleteProject(w http.ResponseWriter, r *http.Request) {
+		http.Error(w, err.Error(), http.StatusInternalServerError)
+		return
+	}
+
+	for i, service := range config.Services {
+		if service.Volume != nil {
+			if _, err := app.Supabase.CreateVolume(serviceTables[i].Id, userId, service.Volume.MountPath, int32(service.Volume.StorageGB)); err != nil {
+				writeError(w, http.StatusInternalServerError, "Couldn't attach the volume. Please try again.", err)
+				return
+			}
+
+			app.Deploy.AttachVolume(r.Context(), deploy.Service{
```

golangci-lint (2.12.2)

[error] Error return value of `app.Deploy.AttachVolume` is not checked

(errcheck)